sklearn/linear_model/tests/test_ridge.py

```diff
@@ -787,3 +791,24 @@ def test_errors_and_values_svd_helper():
 def test_ridge_classifier_no_support_multilabel():
     X, y = make_multilabel_classification(n_samples=10, random_state=0)
     assert_raises(ValueError, RidgeClassifier().fit, X, y)
+
+
+def test_best_score_():
+
+    def fit_and_get_best_score_(reg):
+        reg.fit(X_diabetes,y_diabetes)
+        assert_equal(type(reg.best_score_), np.float64)
```

Comment:
I wonder if we should test this more precisely, e.g. by retraining for the best `alpha` with `cross_val_score`.